src/bp/ui-admin/src/Pages/Workspace/Bots/CreateBotModal.tsx

```diff
@@ -1,5 +1,5 @@
 import { Button, Classes, Dialog, FormGroup, InputGroup, Intent } from '@blueprintjs/core'
-import { BotTemplate } from 'botpress/sdk'
+import { BotTemplate, BotConfig } from 'botpress/sdk'
```

Comment:
Imports should be in alphabetical order.

Reply:
@loicndjoyi Please install [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) and [TSLint](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin) so your editor uses the same styling rules everywhere. It will automatically fix a bunch of stuff, including import order

Reply:
Thank you @allardy. It is installed now.